cmd/cmp/main.go

```diff
@@ -1,12 +1,9 @@
 package cmp
 
 import (
-	"fmt"
-	"github.com/hexops/gotextdiff"
-	"github.com/hexops/gotextdiff/myers"
-	"github.com/hexops/gotextdiff/span"
```

Comment:
Those are still in `go.mod` file where dependencies are listed. Run `go mod tidy` in the project folder then check in the updated `go.mod` and `go.sum` files.
Seach online to understand what those files are for.

Reply:
We should delete stage/diff_test.go because that test was using this deprecated library. After doing that, running `go mod tidy` will get rid of this dependency in the `go.mod` file.